nativeApp/src/volunteer_views/screens/VolunteerHome/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from 'styled-components/native'
+
+import { Heading as H } from '../../../lib/ui/typography';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
```

Comment:
If you're separating local imports from package imports, shouldn't this be in the latter, not the former?